core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1567,14 +1575,15 @@ public Server build() {
             if (!Flags.useOpenSsl() && !SystemInfo.jettyAlpnOptionalOrAvailable()) {
                 throw new IllegalStateException(
                         "TLS configured but this is Java 8 and neither OpenSSL nor Jetty ALPN could be " +
-                        "detected. To use TLS with Armeria, you must either use Java 9+, enable OpenSSL, " +
-                        "usually by adding a build dependency on the " +
-                        "io.netty:netty-tcnative-boringssl-static artifact or enable Jetty ALPN as described " +
+                        "detected. To use TLS with Armeria, you must either use Java 9+, enable " +
+                        "OpenSSL, usually by adding a build dependency on the " +
+                        "io.netty:netty-tcnative-boringssl-static artifact or " +
+                        "enable Jetty ALPN as described " +
```

Comment:
Could you explain why you needed this change?

Reply:
If you are asking about this specific exception then I did not add this code. It is existing code. I can look at why this was added and report back to you.

Reply:
any good news from here? >_<